packages/itwin/tree-widget/src/components/trees/stateless/common/components/BaseTree.tsx

```diff
@@ -50,16 +52,21 @@ interface BaseTreeOwnProps {
   density?: "default" | "enlarged";
   noDataMessage?: ReactNode;
   reportUsage?: (props: { featureId?: UsageTrackedFeatures; reportInteraction: boolean }) => void;
+  textToHighlight?: string;
+  activeMatchIndex?: number;
+  setTotalMatches?: (matches: number) => void;
```

Comment:
I guess we should remove these two props until stepping through result can be fully implemented.

Reply:
Right probably makes sense to remove for now.